sklearn/model_selection/_validation.py

```diff
@@ -24,13 +24,191 @@
+
+        For evaluating multiple metrics, either give a list of (unique) strings
+        or a dict with names as keys and callables as values.
+
+        NOTE that when using custom scorers, each scorer should return a single
+        value. Metric functions returning a list/array of values can be wrapped
+        into multiple scorers that return one value each.
+
+        See :ref:`_multivalued_scorer_wrapping` for an example.
+
+        If None, the estimator's default scorer (if available) is used.
+
+    cv : int, cross-validation generator or an iterable, optional
+        Determines the cross-validation splitting strategy.
+        Possible inputs for cv are:
+          - None, to use the default 3-fold cross validation,
+          - integer, to specify the number of folds in a `(Stratified)KFold`,
+          - An object to be used as a cross-validation generator.
+          - An iterable yielding train, test splits.
+
+        For integer/None inputs, if the estimator is a classifier and ``y`` is
+        either binary or multiclass, :class:`StratifiedKFold` is used. In all
+        other cases, :class:`KFold` is used.
+
+        Refer :ref:`User Guide <cross_validation>` for the various
+        cross-validation strategies that can be used here.
+
+    n_jobs : integer, optional
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs'.
+
+    verbose : integer, optional
+        The verbosity level.
+
+    fit_params : dict, optional
+        Parameters to pass to the fit method of the estimator.
+
+    pre_dispatch : int, or string, optional
+        Controls the number of jobs that get dispatched during parallel
+        execution. Reducing this number can be useful to avoid an
+        explosion of memory consumption when more jobs get dispatched
+        than CPUs can process. This parameter can be:
+
+            - None, in which case all the jobs are immediately
+              created and spawned. Use this for lightweight and
+              fast-running jobs, to avoid delays due to on-demand
+              spawning of the jobs
+
+            - An int, giving the exact number of total jobs that are
+              spawned
+
+            - A string, giving an expression as a function of n_jobs,
+              as in '2*n_jobs'
+
+    return_train_score : boolean, default False
+        Whether to include train scores in the return dict if ``scoring`` is
+        of multimetric type.
+
+        Note that this parameter has no effect for single metric ``scoring``.
+
+    Returns
+    -------
+    scores : array of float or dict of float arrays, array shape=(n_splits,)
+        Array of scores of the estimator for each run of the cross validation.
+
+        For multi-metric evaluation, a dict of arrays containing the score/time
+        arrays for each scorer is returned. The possible keys for this ``dict``
+        are:
+
+            - ``test_scores`` The score array for test scores on each cv split.
```

Comment:
why are these plural? shouldn't the keys be identical, where possible, to *SearchCV? More than that, I'd consider a test that apart from parameters, `cross_validate` remains a slice of *SearchCV output.